plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFunctionFunctions.java

```diff
@@ -40,21 +41,62 @@ private FissionFunctionFunctions()
 
     @ScalarFunction("fission_dnsdb")
     @Description("Converts the string to alternating case")
-    @SqlType(StandardTypes.VARCHAR)
-    public static Slice dnsdb(@SqlType(StandardTypes.VARCHAR) Slice slice)
+    @SqlType(StandardTypes.BIGINT)
+    public static long dnsdb(@SqlType(StandardTypes.VARCHAR) Slice slice) throws JSONException, IOException
     {
         CloseableHttpClient httpClient;
         HttpClientBuilder builder = HttpClientBuilder.create().setDefaultCookieStore(cookieStore);
         httpClient = builder.build();
         String result = "";
+        int count = 0;
         try {
             HttpGet getRequest = new HttpGet(String.format("%s/dnsdb?lookup=%s", FissionFunctionConfigProvider.getFissionFunctionBaseURL(), slice.toStringUtf8()));
             HttpResponse response = httpClient.execute(getRequest);
             result = EntityUtils.toString(response.getEntity());
+            JSONObject myObject = new JSONObject(result);
+
+            if (myObject.has("Message")) {
+                count = 0;
+            }
+            else if (myObject.has("total_count")) {
+                count = Integer.parseInt(myObject.getString("total_count"));
+            }
+        }
+        catch (IOException e) {
+            throw e;
+        }
+        catch (JSONException e) {
+            throw e;
+        }
+        return count;
+    }
+
+    @ScalarFunction("fission_despicablename")
+    @Description("Converts the string to alternating case")
+    @SqlType(StandardTypes.BIGINT)
+    public static long despicablename(@SqlType(StandardTypes.VARCHAR) Slice slice) throws JSONException, IOException
+    {
+        CloseableHttpClient httpClient;
+        HttpClientBuilder builder = HttpClientBuilder.create().setDefaultCookieStore(cookieStore);
+        httpClient = builder.build();
+        String result = "";
+        int count = 0;
+        try {
+            HttpGet getRequest = new HttpGet(String.format("%s/despicablename?domain=%s", FissionFunctionConfigProvider.getFissionFunctionBaseURL(), slice.toStringUtf8()));
+            HttpResponse response = httpClient.execute(getRequest);
+            result = EntityUtils.toString(response.getEntity());
+            JSONObject myObject = new JSONObject(result);
+
+            if (myObject.has("probs")) {
+                count = Integer.parseInt(myObject.getJSONArray("probs").getJSONArray(0).getString(0));
+            }
         }
         catch (IOException e) {
-            result = e.getMessage();
+            throw e;
+        }
+        catch (JSONException e) {
```

Comment:
Could use a multi-catch here! 
```suggestion
        catch (IOException | JSONException e) {
```